libtsuba/include/tsuba/RDGSlice.h

```diff
@@ -74,6 +74,32 @@ class KATANA_EXPORT RDGSlice {
   const std::shared_ptr<arrow::Table>& node_properties() const;
   const std::shared_ptr<arrow::Table>& edge_properties() const;
 
+  // NB: The following interface differs from the one provided by RDG for
+  // property loading/unloading.
+  // For example, this interface uses "remove" instead of "unload" to emphasize
+  // that property modifications will not be persisted (RDGSlice is read-only).
```

Comment:
This sounds confusing to me. "remove" vs. "unload" don't seem to make the distinction that the modifications will/won't be persisted.

Reply:
"remove" vs. "unload" is the distinction that `RDG` currently makes. Remove means "remove from memory + assert not modified", unload means "if necessary, write back update column and update in-memory data structures to point to the new file + remove from memory". I can add comments but I would like to maintain the same verbiage as `RDG`.

Reply:
I added more comments to clarify the intention of the API. But while I was digging around it looked like maybe the `RDG` remove is actually stronger than I thought? It looks like it actually removes the property from the graph, not just from memory. And those are not the semantics of this remove. So maybe we should call these functions unload and the write-only-ness of the interface can just be the lack of an upsert?